Assets/Scripts/Game/UserInterfaceWindows/DaggerfallControlsWindow.cs

```diff
@@ -294,6 +247,8 @@ private void UpdateKeybindButtons()
             SetupKeybindButtons(lookKeys, 32, 36, 269, 102, false);
             SetupKeybindButtons(uiKeys, 36, 40, 269, 147, false);
 
+            currentBindingsButton.Label.Text = ControlsConfigManager.Instance.UsingPrimary ? "Primary" : "Secondary";
```

Comment:
Don't let me forget to add these strings to the Internal_Strings collection for localization later. :)

Reply:
Got it :)